recipes/r-knitcitations/meta.yaml

```diff
@@ -14,7 +14,7 @@ source:
   sha256: 3b366d6b7cd02998d451f820a7883e6c5c315391a5301112a80c8b725969081e
 
 build:
-  merge_build_host: True  # [win]
+  merge_build_host: False  # [win]
```

Comment:
This line should stay as it is, but you can add a line "skip: true  # [win]"

Reply:
thanks for the tips. I fixed it now :+1: